src/main/java/com/valleon/rewardyourteacherapi/configuration/SwaggerConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.valleon.rewardyourteacherapi.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2WebMvc;
+
+@Configuration
+@EnableSwagger2WebMvc
+public class SwaggerConfig {
```

Comment:
Is this all that's needed for the Swagger configuration?

I would later commit a file that's more extensive so you look through it and see if it's preferable.

Reply:
yes thats need for swagger configuratin and sure i will look through it when you will commit a file that is more extensive.